### PR TITLE
Add certifi and extra connect opts for :gun

### PR DESCRIPTION
**Pull request by BunnyDunker**
Added some extra connect opts to try and get rid of the ssl warnings in the logs

### CODE REVIEW

File: lib/slax_web/websocket_listener.ex [hunk 1]
```diff
@@ -26,9 +26,18 @@ defmodule SlaxWeb.WebsocketListener do
...
       :gun.open(:binary.bin_to_list("wss-primary.slack.com"), 443, %{
+        connect_timeout: 60000,
+        retry: 10,
+        retry_timeout: 300,
         transport: :tls,
         protocols: [:http],
-        tls_opts: [verify: :verify_none]
+        http_opts: %{version: :"HTTP/1.1"},
+        tls_opts: [
+          verify: :verify_none,
```
Comment (grossvogel) [inline lib/slax_web/websocket_listener.ex:36]: I'd be surprised if `:verify_none` gets rid of the error... isn't the lack of verification what the error is complaining about?

**Review by grossvogel — COMMENTED** [review]
I'm curious, is this something we can or can't replicate locally?

**Review by grossvogel — DISMISSED** [review]
I'll approve in case you've done more research than I have (entirely possible) and need to get it merged to know for sure

File: lib/slax_web/websocket_listener.ex (hunk 1 above)
Comment (BunnyDunker) [reply lib/slax_web/websocket_listener.ex:36]: Yeah I was following the option for connecting with TLS/SSL but without caring to verify the host, at least at first so like that we don't have to setup a cacerts file

**Comment by BunnyDunker** [comment]
> I'm curious, is this something we can or can't replicate locally?

I have not been able to reproduce this locally

**Review by grossvogel — APPROVED** [review]
If this doesn't work as we expect, I'd encourage spending more time on a local repro... if necessary we can go as far as to build and run the docker image locally, which ought to behave identically to prod if we feed it the right env vars

**Comment by BunnyDunker** [comment]
Yeah looks like I should reproduce it locally, I'll work on figuring out how to do that with docker

**Pull request merged**